### ISSUE TITLE
Consider dropping commission-based recruiting

### ISSUE THREAD

**Issue by doismellburning** [vzaar/note-to-recruiters#11]
Team Prime are the only "recruitment firm" I've ever heard consistently good things about, and their founder Thayer recently published a great article about [commission-based recruiting being inherently flawed](http://team-prime.com/blog/why-commission-based-recruiting-is-inherently-flawed) - this may be of interest.

**Comment by edjames** [comment]
Interesting approach, thanks. I'm not sure if we are ready to make this kind of change though, but certainly something we'll consider and discuss. Thanks.

**Comment by sevitz** [comment]
Hi Kristian

I think Thayer's point is good, but what we've found is working with a single recruiter/jobs board simply isn't enough when recruiting a single dev. No single recruiter has enough market share, that we would be able to find a dev in a timely fashion. If we need to see 15-30 CVs (an arbitrary number) to fill a position, and a good recruiter only sends us 3-4 CVs, it makes it a lot harder to fill that position.

I think that approach works better for larger companies continually recruiting. But for us, I think we just wouldn't see enough candidates to be able to make a hire, unless we got really lucky.

Adrian
